src/clientAuth/bearerHeader.js

```diff
@@ -6,49 +12,47 @@ class BearerHeader extends ClientAuthnMethod {
   }
 
   /**
-   * More complicated logic then I would have liked it to be
-   * @param {*} cis Request class instance
-   * @param {*} ci Client information
-   * @param {*} requestArgs: request arguments
-   * @param {*} httpArgs HTTP header arguments
-   * @param {*} kwargs
+   * @param {?ResourceRequest} cis Request class instance
+   * @param {?ClientInfo} ci Client information
+   * @param {?Object.<string, string>} requestArgs Request arguments
+   * @param {?Object.<string, string>} httpArgs HTTP header arguments
+   * @param {?Object.<string, string>} kwargs Other optional arguments
    */
   construct(cis, cliInfo, httpArgs, kwargs) {
     cis = cis || null;
```

Comment:
You can use default parameter instead of this. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters